libs/designer/src/lib/ui/templates/__tests__/displayConnections.spec.tsx

```diff
@@ -13,31 +13,66 @@ import React from 'react';
 describe('ui/templates/workflowconnections', () => {
   let store: AppStore;
   let templateSliceData: TemplateState;
-  let template1Manifest: Template.Manifest;
+  let template1Manifest: Template.TemplateManifest;
+  let workflow1Manifest: Template.WorkflowManifest;
   let param1DefaultValue: string;
   let param2DefaultValue: string;
 
   beforeAll(() => {
     param1DefaultValue = 'default value for param 1';
     param2DefaultValue = 'boolean';
     template1Manifest = {
+      id: 'template1Manifest',
       title: 'Template 1',
-      description: 'Template 1 Description',
+      summary: 'Template 1 Description',
       skus: ['standard', 'consumption'],
+      workflows: {
+        default: { name: 'default' },
+      },
+      details: {
+        By: '',
+        Type: '',
+        Category: '',
+      },
+      artifacts: [
+        {
+          type: 'description',
+          file: 'description.md',
```

Comment:
Why are we not keeping this empty in test files... is something breaking? also artifacts was not a required property right so if not present we should not fail in our code